src/style.css

```diff
@@ -1 +1,70 @@
 /* add styles here */
+html {
+  --active-star: url(/src/images/star-active.svg);
+  --star: url(/src/images/star.svg);
```

Comment:
You can set a default image by default to the `stars__star `class and change it using nth-chilt rule